api/src/main/java/uk/gov/dhsc/htbhf/claimant/converter/ClaimToClaimResponseDTOConverter.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.dhsc.htbhf.claimant.converter;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimResponseDTO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Component
+@AllArgsConstructor
```

Comment:
i don't think it is required to have AllArgsConstructor here as class doesn't have any members of another class